### PR TITLE
feat: tabbed display of subscription details

### PR DESCRIPTION
**Pull request by gertvangravity**
- toggles to switch between tree and tabview (tab is default).
- tab block for generic details with fixed height
- tabbed display of service config details
- small styling (indentation) fix

### CODE REVIEW

File: src/components/subscriptionDetail/SubscriptionInstance.tsx [hunk 1]
```diff
@@ -18,12 +18,7 @@ import { isArray, partition } from "lodash";
 import React from "react";
 import { FormattedMessage } from "react-intl";
 
-interface ISubscriptionInstance {
-    subscription_instance_id: string;
-    name: string;
-    label?: string;
-    [index: string]: any;
-}
+import { ISubscriptionInstance } from "../../utils/types";
```
Comment (tjeerddie) [inline src/components/subscriptionDetail/SubscriptionInstance.tsx:21]: you don't need the `../../`

File: src/components/subscriptionDetail/TabView.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+export const SUBSCRIPTION_VIEWTYPE_SELECTOR = "subscription-viewtype";
```
Comment (tjeerddie) [inline src/components/subscriptionDetail/TabView.tsx:1]: Rename the file to include `types` or maybe move the contents to `utils/types`?

File: src/components/subscriptionDetail/SubscriptionInstance.tsx (hunk 1 above)
Comment (gertvangravity) [reply src/components/subscriptionDetail/SubscriptionInstance.tsx]: 👍 I noticed I used relative paths in other imports as well. I cleaned those up as well.

File: src/components/subscriptionDetail/TabView.tsx (hunk 2 above)
Comment (gertvangravity) [reply src/components/subscriptionDetail/TabView.tsx]: Ended up moving the contents to `utils/types`

**Review by pboers1988 — APPROVED** [review]
LGTM

**Review by tjeerddie — APPROVED** [review]

**Pull request merged**